File: Linguagem de Programação I/Extras/Beecrowd_Extra/1047.php
<?php

$times = explode(" ", trim(fgets(STDIN)));
list($sh, $sm, $eh, $em) = array_map('intval', $times);

$st = $sh * 60 + $sm;
$et = $eh * 60 + $em;
$d = $et - $st;
if ($d <= 0) {
    $d += 24 * 60;
}
$h = intdiv($d, 60);
$m = $d % 60;
printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n", $h, $m);

?>

File: Linguagem de Programação I/Extras/Beecrowd_Extra/1047.c
#include <stdio.h>

int main() {
    int sh, sm, eh, em;
    scanf("%d %d %d %d", &sh, &sm, &eh, &em);
    int st = sh * 60 + sm;
    int et = eh * 60 + em;
    int d = et - st;
    if (d <= 0) {
        d += 24 * 60;
    }
    int h = d / 60;
    int m = d % 60;
    printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n", h, m);
    return 0;
}